taichi/codegen/codegen_llvm.cpp

```diff
@@ -1187,7 +1213,8 @@ void CodeGenLLVM::visit(SNodeLookupStmt *stmt) {
   } else if (snode->type == SNodeType::dense ||
              snode->type == SNodeType::pointer ||
              snode->type == SNodeType::dynamic ||
-             snode->type == SNodeType::bitmasked) {
+             snode->type == SNodeType::bitmasked ||
+             snode->type == SNodeType::bit_struct) {
```

Comment:
I add this line because when I run script `test_bit_struct.py`, the visiting of `SNodeLookUpStmt` will crash. I might not use a right method to handle this problem, but I think we can ignore this for now, I will take care of it after the `GetChStmt`,`GlobalLoadStmt` and `GlobalStoreStmt` are right.

Reply:
Actually for `bit_struct`, you can simply return `parent` as the `llvm_val`.

Reply:
(We may need a `BitArrayMeta` later for bitmasked bit arrays, but for now and for `BitStruct`, we can simply stick to the simple solution above)